dlme_airflow/harvester/source_harvester.py

```diff
@@ -14,4 +14,9 @@ def data_source_harvester(**kwargs):
 
     collection = kwargs.get("collection")
     # dataframe_to_file(source, collection.provider.name, collection)
-    return dataframe_to_file(collection)
+    df_and_csv = dataframe_to_file(collection)
+    df = df_and_csv["source_df"]
+    working_csv = df_and_csv["working_csv"]
+    dataframe_stats = {"record_count": df.shape[0]}
```

Comment:
I think this should be df.shape[0]-1 since it's counting the header row as well which is not included in the transformed ndjson output.